### ISSUE TITLE
Accessibility issue: Elements must meet minimum color contrast ratio thresholds on /en/latest/examples/autodoc.html

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/sphinx_wagtail_theme#329]
## What
An accessibility scan flagged 36 elements on https://sphinx-wagtail-theme.readthedocs.io/en/latest/examples/autodoc.html because elements must meet minimum color contrast ratio thresholds. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=playwright.

The following elements need attention:

- `<span class="versionmodified added">Added in version 0.6.</span>` (selector: `.method:nth-child(3) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 0.6.</span>` (selector: `.method:nth-child(4) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="pre">&apos;env&apos;</span>` (selector: `.versionchanged:nth-child(5) > p > .literal.docutils:nth-child(4) > .pre`)
- `<span class="versionmodified added">Added in version 1.4: </span>` (selector: `.versionadded:nth-child(6) > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 7.4: </span>` (selector: `.method:nth-child(6) > dd > .versionadded:nth-child(7) > p > .added.versionmodified`)
- `<p class="admonition-title">See also</p>` (selector: `.method:nth-child(8) > dd > .admonition.seealso > .admonition-title`)
- `<span class="versionmodified added">Added in version 1.0.</span>` (selector: `.method:nth-child(8) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="pre">alternate</span>` (selector: `.method:nth-child(8) > dd > .versionchanged:nth-child(9) > p > .literal.docutils:nth-child(2) > .pre`)
- `<span class="pre">title</span>` (selector: `.versionchanged:nth-child(9) > p > .literal.docutils:nth-child(3) > .pre`)
- `<span class="pre">False</span>` (selector: `.versionchanged:nth-child(9) > p > .literal.docutils:nth-child(7) > .pre`)
- `<span class="pre">app.add_stylesheet()</span>` (selector: `.versionchanged:nth-child(10) > p > .literal.docutils > .pre`)
- `<span class="versionmodified added">Added in version 1.0.</span>` (selector: `.method:nth-child(10) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 1.0.</span>` (selector: `.method:nth-child(11) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 1.4.</span>` (selector: `.method:nth-child(12) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 1.6.</span>` (selector: `.method:nth-child(13) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 0.6.</span>` (selector: `.method:nth-child(15) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 1.8.</span>` (selector: `.method:nth-child(16) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 1.6.</span>` (selector: `.method:nth-child(17) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 1.0.</span>` (selector: `.method:nth-child(18) > dd > .versionadded > p > .added.versionmodified`)
- `<p class="admonition-title">See also</p>` (selector: `.method:nth-child(19) > dd > .admonition.seealso > .admonition-title`)
- `<span class="versionmodified added">Added in version 0.5.</span>` (selector: `.method:nth-child(19) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="pre">app.add_javascript()</span>` (selector: `.method:nth-child(19) > dd > .versionchanged:nth-child(9) > p > .literal.docutils > .pre`)
- `<span class="versionmodified added">Added in version 1.3.</span>` (selector: `.method:nth-child(20) > dd > .versionadded:nth-child(4) > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 3.1: </span>` (selector: `.method:nth-child(20) > dd > .versionadded:nth-child(5) > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 0.6.</span>` (selector: `.method:nth-child(21) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 1.8.</span>` (selector: `.method:nth-child(22) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 1.0.</span>` (selector: `.method:nth-child(27) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 1.1.</span>` (selector: `.method:nth-child(28) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 1.4.</span>` (selector: `.method:nth-child(29) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 1.8.</span>` (selector: `.method:nth-child(30) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 9.1.</span>` (selector: `.method:nth-child(31) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 0.5.</span>` (selector: `.method:nth-child(36) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="versionmodified added">Added in version 1.0.</span>` (selector: `.method:nth-child(39) > dd > .versionadded > p > .added.versionmodified`)
- `<span class="pre">(major,</span>` (selector: `.method:nth-child(39) > dd > .versionchanged > p > .literal.docutils > .pre:nth-child(1)`)
- `<span class="pre">minor)</span>` (selector: `.versionchanged > p > .literal.docutils > .pre:nth-child(2)`)
- `<span class="versionmodified added">Added in version 1.3.</span>` (selector: `.method:nth-child(41) > dd > .versionadded > p > .added.versionmodified`)

To fix this, ensure the contrast between foreground and background colors meets wcag 2 aa minimum contrast ratio thresholds.

Specifically:

Fix any of the following:
-   Element has insufficient color contrast of 4.41 (foreground color: #ffffff, background color: #a456c9, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
